ament_cmake_pytest/cmake/ament_add_pytest_test.cmake

```diff
@@ -98,6 +98,46 @@ function(ament_add_pytest_test testname path)
     list(APPEND cmd "-We")
   endif()
 
+  option(AMENT_CMAKE_PYTEST_WITH_COVERAGE
+    "Generate coverage information for Python tests"
+    OFF)
+
+  # enable pytest coverage automatically if the package test_depends on pytest_cov
+  if("python3-pytest-cov" IN_LIST ${PROJECT_NAME}_TEST_DEPENDS)
+    set(AMENT_CMAKE_PYTEST_WITH_COVERAGE ON)
```

Comment:
I think this condition should only be used to populate the default value of the option. Otherwise a user can't set the option to `OFF` if the package has a test dependency on `python3-pytest-cov`. Like this:

```
if("python3-pytest-cov" IN_LIST ${PROJECT_NAME}_TEST_DEPENDS)
  set(default ON)
else()
  set(default OFF)
endif()
option(AMENT_CMAKE_PYTEST_WITH_COVERAGE
  "Generate coverage information for Python tests"
  ${default})
```

Reply:
Done in 33c00d029e893bd91513ed6dd925793e74e3fcd7.